src/xrGame/DiscordRPC.h

```diff
@@ -0,0 +1,21 @@
+#pragma once
+
+class DiscordRPC
+{
+private:
+    int64_t     start_time          {};
+    string64    build_name          {};
+    string64    current_level_name  {};
+    pcstr       current_task_name   {};
```

Comment:
Currently, we can make `current_level_name` to be local variable instead of member variable. :)
```suggestion
    int64_t start_time{};
    string64 build_name{};
    pcstr current_task_name{};
```

Reply:
I agree. We can set all properties related to level inside `if` statement, when new level is passed.

Reply:
Did not make variable local, because then current_task_name overwrites location in status.